code/game/objects/effects/glowshroom.dm

```diff
@@ -153,8 +153,17 @@
 
 /obj/structure/glowshroom/attackby(obj/item/I, mob/user)
 	..()
+	var/force = I.force

```

Comment:
same as the one used in the [spacevine attackby()](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/events/spacevine.dm#L488)

I didn't care for it myself, to be honest.